libcaf_openssl/src/manager.cpp

```diff
@@ -66,6 +81,13 @@ void* manager::subtype_ptr() {
   return this;
 }
 
+bool manager::authentication_enabled() {
+  auto& cfg = system().config();
+  return cfg.openssl_certificate.size() || cfg.openssl_key.size()
```

Comment:
`cfg.openssl_certificate.size() > 0 || ...` is a tiny bit more verbose but a whole lot better to read. :)